src/rule_engine/tests/test_rule_engine.py

```diff
@@ -231,3 +231,7 @@ def test_iin_value_error(data: dict[str, t.Any], condition_value: t.Any) -> None
     with pytest.raises(ValueError):
         rule = Rule(foo__iin=condition_value)
         evaluate(rule, data)
+
+
+def test_regression_iin_field_not_in_data() -> None:
```

Comment:
Not a regression, it was expected behaviour; add test in line 27 instead:

        (Operator.IIN, None, "Hello world", False),

Reply:
71b946affb72bf11904716de8c4847e8499b99a2